### PR TITLE
Account edit

### PR DESCRIPTION
**Pull request by JamesNix**
This feature so far allows user to update the email and password of this account. But edge cases have not all been tested for and some precautions must be added to prevent unintended changes. More testing is required.

### CODE REVIEW

File: app/views/devise/confirmations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (elizamarcum) [inline app/views/devise/confirmations/new.html.erb]: This file isn't related to your changes.

File: app/views/devise/passwords/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<h2>Change your password</h2>
```
Comment (elizamarcum) [inline app/views/devise/passwords/edit.html.erb]: This should be converted to haml.

Comment (elizamarcum) [inline app/views/devise/passwords/edit.html.erb]: Actually, I don't think this file is related to your functionality.

File: app/views/devise/passwords/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<h2>Forgot your password?</h2>
```
Comment (elizamarcum) [inline app/views/devise/passwords/new.html.erb]: I don't think this file is related to your changes.

File: app/views/devise/registrations/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<h2>Sign up</h2>
```
Comment (elizamarcum) [inline app/views/devise/registrations/new.html.erb]: This is the erb version of a file that is present in haml.  It should be deleted.

File: app/views/devise/sessions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<h2>Sign in</h2>
+
+<%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```
Comment (elizamarcum) [inline app/views/devise/sessions/new.html.erb]: I don't think this file is related to your changes.

File: app/views/devise/shared/_links.erb [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
```
Comment (elizamarcum) [inline app/views/devise/shared/_links.erb]: This is the erb version of a file that we already converted to haml.  It should be deleted.

**Comment by JohnOfTheWater** [comment]
Pull request updated with changes to reflect Eliza's suggestions.

**Comment by elizamarcum** [comment]
@JohnOfTheWater @JamesNix There still seems to be a stray view file in the diff.

**Comment by JamesNix** [comment]
@elizabrock We've removed stray file along with some whitespace.

cc: @JohnOfTheWater

**Pull request merged**